### DISCUSSION TITLE
windows Start white screen

### DISCUSSION THREAD

**Discussion by 903531306** [bdlukaa/fluent_ui#776] [Q&A]
Hi, ask me a question. I see Windows fluent_ The UI project didn't start with a white screen, but I looked at the startup code and didn't see where it was set. My project started with a white screen before the financial interface. I want to ask how to deal with it, because there are few reference materials and I don't know how to deal with it. Thank you

**Comment by bdlukaa** [comment]
If your project was created on some recent versions, the window should be displayed after the first frame is rendered.

https://github.com/flutter/flutter/issues/41980
